s2/bq/sql/VERSION.sql

```diff
@@ -0,0 +1,14 @@
+-----------------------------------------------------------------------
+--
+-- Copyright (C) 2021 CARTO
+--
+-----------------------------------------------------------------------
+
+CREATE OR REPLACE FUNCTION `@@BQ_PROJECTID@@.@@BQ_DATASET_S2@@.VERSION`()
+    RETURNS INT64
```

Comment:
Since this is tied up to a `s2-geometry.js` release I would rather return a string with the release of that library. Personal preference.